index.js

```diff
@@ -0,0 +1,18 @@
+// Дана строка. Вернуть длину самого короткого слова
+// Пример:
+// getLengthOfShortestWord(‘Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.’) 	// => 2
+
+//O(2n)
+function getLengthOfShortestWord(str) {
+  return str
+    .split(" ")
```

Comment:
you also consider here '.' and ',' to the word's part. Let's get rid of them. In case of: 'word a, adas.' result will be 2, but expected 1

Reply:
add avoiding symbols